src/app/features/notifications/notifications.component.html

```diff
@@ -0,0 +1,91 @@
+<button
+  mat-icon-button
+  matTooltip="Notifications"
+  i18n-matTooltip="Notifications"
+  [matMenuTriggerFor]="data"
+  (menuOpened)="onNotificationBellClick()"
+>
+  <span
+    [matBadge]="
+      hasUnreadNotificationCount > 0 ? hasUnreadNotificationCount : null
+    "
+    matBadgeColor="accent"
+  >
+    <fa-icon class="white" icon="bell"></fa-icon>
+  </span>
+</button>
+
+<mat-menu #data>
+  <div class="notification-panel">
+    <div
+      class="flex-row justify-space-between align-center notification-panel-header"
+    >
+      <h1 class="notification-title" i18n>Notifications</h1>
+
+      <button
+        mat-icon-button
+        [matMenuTriggerFor]="settingsMenu"
+        (click)="$event.stopPropagation()"
+        icon-button
+      >
+        <fa-icon icon="ellipsis"></fa-icon>
+      </button>
+    </div>
+
+    <!-- Tabs -->
+    <mat-tab-group
+      [(selectedIndex)]="selectedTab"
+      (click)="$event.stopPropagation()"
+    >
+      <mat-tab label="All" i18n-label></mat-tab>
+      <mat-tab label="Unread" i18n-label></mat-tab>
+    </mat-tab-group>
+
+    <ng-container *ngIf="selectedTab === 0">
+      <!-- All Notifications -->
+      <app-notification-item
+        *ngFor="let notification of allNotifications"
+        [notification]="notification"
+        [onMarkAsRead]="markAsRead"
+        [onDeleteNotification]="deleteNotification.bind(this, notification)"
+      ></app-notification-item>
+      <div
+        class="flex-column no-notification-message"
+        *ngIf="allNotifications.length === 0"
+      >
+        <fa-icon class="no-notification-icon" icon="bell"></fa-icon>
+        <span class="no-notification-text" i18n>You have no notifications</span>
+      </div>
+    </ng-container>
+
+    <ng-container *ngIf="selectedTab === 1">
```

Comment:
rather than copying the (exactly same?) template, we can probably do something like `let ... of (selectedTab === 1 ? unreadNotifications : allNotifications)`?

Reply:
Thanks for this suggestion. I have update the code accordingly. Please take a look once.